### PR TITLE
popup_task

### PR DESCRIPTION
**Pull request by SamVal007**
### HTML-CSS-Popup

[Demo](https://samval007.github.io/kottans-frontend/HTML-CSS-Popup/)
[Code base](https://github.com/SamVal007/kottans-frontend/tree/main/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/SamVal007/HTML-CSS-Popup/style.css [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    width: 35px;
+    height: 35px;
+    object-fit: contain;
+    margin: 0;
+}
+
+.icon {
+    margin-right: 20px;
+    border: 1px solid transparent;
+}
```
Comment (lenadgit) [inline submissions/SamVal007/HTML-CSS-Popup/style.css:194]: If you see this symbol at Github at the end of the file, it means that the empty line is necessary. 
empty line
Could you please add the empty line under the bracket?

File: submissions/SamVal007/HTML-CSS-Popup/index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+            </ul>
+
+        </nav>
+
+    </header>
+
+
+</body>
+
+</html>
```
Comment (lenadgit) [inline submissions/SamVal007/HTML-CSS-Popup/index.html:127]: If you see this symbol at Github at the end of the file, it means that the empty line is necessary. 
empty line
Could you please add the empty line under the bracket?

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Review by lenadgit — CHANGES_REQUESTED** [review]
Thanks for submission. Good job. 
Could you please format your files by auto-formatting? (to remove unnecessary spaces)

**Review by lenadgit — CHANGES_REQUESTED** [review]
while the user moves on menu icons by Tab key, the popup's icon does not have a focus. It is not clear for the user about further navigation by this icon. Could you please add the focus to this icon
https://nimb.ws/JvR8eF

File: submissions/SamVal007/HTML-CSS-Popup/style.css [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    margin-top: 20px;
+}
+
+.more-open {
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    color: #4a4d4f;
+    flex-wrap: nowrap;
+    margin-left: -10px;
+    margin-bottom: 20px;
```
Comment (lenadgit) [inline submissions/SamVal007/HTML-CSS-Popup/style.css:127]: could you please add this rule for the cursor:
`cursor: pointer;`

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/SamVal007/HTML-CSS-Popup/index.html (hunk 2 above)
Comment (SamVal007) [reply submissions/SamVal007/HTML-CSS-Popup/index.html:127]: Yep! 
Thank You for review!

File: submissions/SamVal007/HTML-CSS-Popup/style.css (hunk 3 above)
Comment (SamVal007) [reply submissions/SamVal007/HTML-CSS-Popup/style.css:122]: Done

**Review by lenadgit — APPROVED** [review]

**Pull request merged**